runtime-testsuite/test/org/antlr/v4/test/runtime/go/BaseTest.java

```diff
@@ -464,13 +516,17 @@ private String locateGo() {
 		return prop;
 	}
 
-	private String locateRuntime() {
+	private static File locateRuntime() {
 		final ClassLoader loader = Thread.currentThread().getContextClassLoader();
 		final URL runtimeSrc = loader.getResource("Go");
```

Comment:
How did you know there was a "Go" resource, and what it was defined as? Is it `"github.com/pboyer/antlr4/runtime/Go"`?